prod_gen3_test/domu.xml

```diff
@@ -8,10 +8,11 @@
 
     <default remote="github" sync-j="10" sync-c="true" />
 
-    <project name="renesas-rcar/meta-renesas" path="meta-renesas" upstream="krogoth" revision="7acbf5e2f99c59478adbc73c6a40d314589a3009" />
-    <project remote="yoctoproject" name="poky" path="poky" upstream="krogoth" revision="krogoth" />
-    <project remote="openembedded" name="meta-openembedded" path="meta-openembedded" upstream="krogoth" revision="55c8a76da5dc099a7bc3838495c672140cedb78e" />
+    <project name="renesas-rcar/meta-renesas" path="meta-renesas" upstream="rocko" revision="d50e055494cea5de0a348c15ee27cb67a3542e0b" />
+    <project remote="yoctoproject" name="meta-virtualization" path="meta-virtualization" upstream="rocko" revision="rocko" />
```

Comment:
why  do we need meta-virtualization support in domu?

Reply:
actually we don`t... removed.